src/transformers/models/whisper/english_normalizer.py

```diff
@@ -14,17 +14,10 @@
 # limitations under the License.
 
 import re
+import unicodedata
```

Comment:
I am guessing we already have this as a dependency?

Reply:
It got moved by `isort` that's it.